lib/config.js

```diff
@@ -13,33 +13,33 @@ let COFFEE_SCRIPT_AVAILABLE = false
 let LIVE_SCRIPT_AVAILABLE = false
 let TYPE_SCRIPT_AVAILABLE = false
 
-// Coffee is required here to enable config files written in coffee-script.
+// Coffee is required here to enable config files written in coffeescript.
 // It's not directly used in this file.
 try {
-  require('coffee-script').register()
+  require('coffeescript').register()
```

Comment:
Now it duplicates L25. It was `coffee-script` here, because there was two npm packages: 
https://www.npmjs.com/package/coffee-script
https://www.npmjs.com/package/coffeescript

`coffee-script` is deprecated, so I think we can remove L18-L22